lib/web_ui/test/text_editing_test.dart

```diff
@@ -435,6 +435,13 @@ void testMain() {
       textEditing = null;
     });
 
+    test('TextInput.requestAutofill', () async {
+      final MethodCall requestAutofill = MethodCall('TextInput.requestAutofill');
+      sendFrameworkMessage(codec.encodeMethodCall(requestAutofill));
+
+      //No-op and without crashing.
```

Comment:
We could check that the `spy` intercepted a response, which would prove that the engine did process the message. For example:

```
      expect(spy.messages, hasLength(1));
```